app/views/users/mod_delete.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :title, "Moderator Tools: #{rtl_safe_username(@user)}" %>
+
+<%= render 'tabs', user: @user %>
+
+<div class="grid">
+  <%= render 'shared/user_mod_sidebar', user: @user %>
+
+  <div class="grid--cell is-9">
+    <h1>Delete or Destroy Account</h1>
+
+    <p>Some users are just blatant spammers or trolls and some users are just unwilling to follow site rules, even after repeated warnings and suspensions. As a moderator, you may delete the user account in these cases.</p>
+
+    <div class="notice is-danger">
+        <p><strong>Take care!</strong> These actions may not be reversible and you will not be asked to confirm after initiating an action.</p>
+    </div>
+
+    <div class="widget">
+        <div class="widget--body">
+            <h4>Destroy Acccount</h4>
+            <p>Destroy Accounts of blatant spammers and trolls.</p>
+
+            <%= link_to 'Destroy Account', destroy_user_path(@user.id), remote: true,
+                        method: :delete, class: 'js-destroy-user button is-danger is-filled' %>
+        </div>
```

Comment:
discuss: We should review whether we actually want this here. Destroying accounts completely is a good way to break the database, and now that we have soft-deletes it's probably not needed.

Reply:
I propose to remove "destroy" for the reasons Art said.

Reply:
@trichoplax if the destroy option is still there, could you remove it from the UI at least?  Thanks.

Reply:
1. Yes, remove the "Destroy Account" button and keep the other three.
2. I defer to @luap42  but I think this is right.
3. I defer to luap42 again but yeah, that sounds better for a public platform rather than our own instance.  I believe "feed to STAT" is delete+ spam-block.